src/screens/DiscoveryScreenV2/fragment/UsersFragment.js

```diff
@@ -171,8 +171,8 @@ const UsersFragment = ({
         <DomainList
           key={`${key}-${index}`}
           onPressBody={() => handleOnPress(item.user || item)}
-          handleSetFollow={() => handleFollow(from, true, item.user || item, index)}
-          handleSetUnFollow={() => handleFollow(from, false, item.user || item, index)}
+          handleSetFollow={() => handleFollow(from, true, item.user || item)}
+          handleSetUnFollow={() => handleFollow(from, false, item.user || item)}
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `handleSetFollow` and `handleSetUnFollow` functions no longer receive the `index` parameter. Ensure that this change doesn't affect their functionality or any other parts of your code that rely on these functions.
```diff
- handleSetFollow={() => handleFollow(from, true, item.user || item, index)}
- handleSetUnFollow={() => handleFollow(from, false, item.user || item, index)}
+ handleSetFollow={() => handleFollow(from, true, item.user || item)}
+ handleSetUnFollow={() => handleFollow(from, false, item.user || item)}
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->